src/relay/op/annotation/annotation.cc

```diff
@@ -201,5 +201,7 @@ TVM_REGISTER_GLOBAL("relay.op.annotation._make.compiler_end")
       return Call(op, {expr}, Attrs(attrs), {});
     });
 
+TVM_DLL extern const Op stop_fusion_op = Op::Get("annotation.stop_fusion");
```

Comment:
Quick question: why are we using external linkage while having an initial value?